platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java

```diff
@@ -102,6 +103,19 @@ default void preHandle(final Event event) {}
      */
     void handleConsensusRound(final Round round, final SwirldDualState swirldDualState);
 
+    /**
+     * Update the stake of the nodes in the address book
```

Comment:
updating Javadoc to be more detailed to clarify purpose and why a default implementation exists.  It is to prevent compilation failure where other parties are responsible for implementing the method.